src/main.py

```diff
@@ -72,17 +72,10 @@ def execute(node_id, op: Op):
 
 def main(args):
 
-    # determine nodes (of type single item node) and operations for the cluster
-    # TODO: Add more fine-grained control over the exact topology and number of nodes
-    if args.num_nodes == 3:
-        nodes = basecases.bootstrap_dependencies_three_nodes()
-    elif args.num_nodes == 6:
-        nodes = basecases.bootstrap_dependencies_six_nodes()
-    elif args.num_nodes == 7:
-        nodes = basecases.bootstrap_dependencies_seven_nodes()
-    else:
-        log.error("%d node count not supported by any demo/test scenerio", args.num_nodes)
-        nodes = None
+    log.info("""Nodes are being created through bootstrap_ranodom_dag() with %s number of types, 
```

Comment:
single " is fine here. Logging should avoid mentioning actual function names. The message can be as simple as "Creating random DAG with %d types of items, with %d nodes per type and cluster complexity %s"

also, this log should be at the beginning of the bootstrap_random_dag function instead of in main()

Reply:
ok done